script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -335,175 +406,196 @@ His screams reverberate throughout the entire room.
+// oopsie woopsie!! uwu I made a fucky wucky!! A wittle fucko boingo!
+"My bad, my bad. He just said something so hilarious that I got a little carried away.
 }
 [sha:0236a351e56f48d2a917c6938bbb5d09eca5c5a8]{
 -- Page 453, Offset 19400. Glued. Mods: @x.
 -- 安心してみんな。今ので満足しちゃったし、あなたたちは適当で済ませるから。手抜き作業でごめんなさいね」
-I'm satisfied now and I'll be dealing with you in the proper way. So don't worry, guys. Sorry for the sloppy work."
+ But don't worry, everyone.
+ I've had my fill now, so I'll be sure to deal with the rest of you by the book.
+ Sorry for the sloppy work."
 }
 [sha:9b43c70ad230851bd4deb3994239a9c6c7896487]{
 -- Page 454, Offset 19401.
 -- 媚びるように甘く微笑むシスター。
-The sister smiles at them flirtatiously and sweetly.
+The nun flashes them a sweetly flirtatious smile.
 }
 [sha:5626b3f61ab0315b08e4fea30f9983cbfc2c8826]{
 -- Page 454, Offset 19402.
 -- それが最後の引き金になった。
-That was the final trigger.
+This is the final straw.
 }
 [sha:a98c6bfaf8202ea9df58daac0f7dfd4cd1838775]{
 -- Page 454, Offset 19403.
 -- 残った獲物たちは金切り声をあげ、結界によって手足を溶かされながらも狂ったように壁を叩く。
-The remaining prey madly pounded on the walls while letting out piercing shrieks, their hands and feet liquefied by the Barrier.
+The remaining prey start shrieking and punching and kicking the walls,
+ even as the Barrier begins to melt their hands and feet.
 }
 [sha:4efcfe4bbe571c58f1911c7c20a32c0390bd43e5]{
 -- Page 454, Offset 19404.
 -- ここから出してくれ、と全身で叫ぶように。
-They screamed with their entire beings, begging to be released.
+They scream with every fiber of their beings,
+ begging to be released.
 }
 [sha:74ec8b06640c451c79f29369e73aa4ae0346bcd3]{
 -- Page 455, Offset 19405.
 -- “やだ、やだ、やだ、やだ……！　お願い、殺さないで、殺さないで、殺さないでください、神さま……！”
-'No, no, no, no...! Please, don't kill me, don't kill me, please don't kill me! Oh God...!'
+"No, no, no, no...! Please, don't kill me, don't kill me, please don't kill me! Oh God...!"
 }
 [sha:70c0962097732b6caaa2a581abf9ad24b6b36320]{
 -- Page 455, Offset 19406.
 -- “お願いします、お願いします、お願いします。なんでもします、なんでもしますから、どうかワタシだけは”
-'Please, please, please. I'll do anything, I'll do anything, so please, leave me, just me―――'
+"Please, please, please. I'll do anything, I'll do anything, so please, spare me, just me―――"
 }
 [sha:0882dbc67aaf31a9a6cf9fa4de15b28a6a73790c]{
 -- Page 456, Offset 19407.
 -- “帰る……今度こそちゃんとお家に帰るからぁ……助けてようお母さん……お母さん、お母さん、お母さ”
 // lint-off:LintNameMisspellings
-'I'll go back... This time for real I'll go back home... Help me mom... Mom... Mom... Mo―――'
+"I'll go back... I mean it this time, I'll go back... Help me mom... Mom... Mom... Mo―――"
 }
 [sha:0e37fd4241b0011518d9e9257c7aecf18df9670b]{
 -- Page 456, Offset 19408.
 -- “待ってよ、アタシはまだ誰も食べてない！　昨日コイツらに襲われてまだ吸血鬼にもなってないのに、まだ一つもいい目にあってないのにこんなのってな……！”
-'Wait, I haven't eaten anyone! Until those guys attacked me yesterday, I wasn't even a vampire, I haven't even scored yet, so why is this happening...!
+// Is this interpretation of いい目にあう correct? I couldn't find anything to back it up
```

Comment:
dunno about "scored" (since this reads like he was cruising to get laid, when i assume he means getting to suck some sweet sweet blood), but I think the interpretation that he hasn't even "gotten to the good part" of being a vampire yet is right